test/src/unit-ReadCellSlabIter.cc

```diff
@@ -72,7 +72,7 @@ struct ReadCellSlabIterFx {
   template <class T>
   void create_result_space_tiles(
       const std::vector<tdb_shared_ptr<FragmentMetadata>>& fragments,
-      const Domain* dom,
+      shared_ptr<const Domain> dom,
```

Comment:
looks good here. although there are some debates between shared_ptr<const Domain> and const shared_ptr<Domain>&, shared_ptr<const Domain> is preferred.